docs/usershowcase-template.rst

```diff
@@ -0,0 +1,6 @@
+User showcase
+*************
+This page contains example notebooks demonstrating how CIL users have solved a variety of interesting problems using CIL tools. Click below to see the rendered notebooks, or access and run the code yourself from https://github.com/TomographicImaging/CIL-User-Showcase/tree/main 
+
```

Comment:
Should we add a note here about how each notebook has been tested on a version of CIL specified in the notebook and not guaranteed to run on the latest CIL release?